docs/user_guide.md

```diff
@@ -70,14 +70,26 @@ You can create a project after you signed in. Check on the "Access Level" checkb
 
 ![create project](img/new_create_project.png)  
 
-After the project is created, you can browse repositories, users and logs using the navigation tab.  
+After the project is created, you can browse repositories, members, logs, replication and configuration using the navigation tab.  
 
 ![browse project](img/new_browse_project.png)  
 
 All logs can be listed by clicking "Logs". You can apply a filter by username, or operations and dates under "Advanced Search".  
 
 ![browse project](img/new_project_log.png)  
 
+Configure the project properties by clicking "Configuration".
+
+* If you want to make all repositories under the project public, select the `Public` checkbox.
+
+* If you want to prevent unsigned images under the project from being pulled, select the `Enable content trust` checkbox.
+
+* If you want to prevent vulnerable images under the project from being pulled, select the `Prevent vulnerable images from running` checkbox and change the severity level of vulnerabilities. Images cannot be pulled if their level equals the currently selected level or higher.

```

Comment:
To prevent vulnerable images under the project from being pulled, select the `Prevent vulnerable images from running` checkbox and change the severity level of vulnerabilities. Images cannot be pulled if their level equals to or higher than the currently selected level.